modules/programs/chromium.nix

```diff
@@ -112,9 +112,13 @@ let
         google-chrome-beta = "Google/Chrome Beta";
         google-chrome-dev = "Google/Chrome Dev";
         brave = "BraveSoftware/Brave-Browser";
+        vivaldi = "vivaldi";
```

Comment:
Is this necessary?

Reply:
It's not, but I left it in for consistency with the other browsers. Should I remove it?

Reply:
Yeah, I think it's best to only record differences from the default value (like in `linuxDirs` below).